cms/utils/__init__.py

```diff
@@ -16,15 +21,42 @@ def get_current_site():
     return Site.objects.get_current()
 
 
+@lru_cache(512)
+def get_object_language(request):
+    try:
+        resolved = resolve(request.path)
+    except Resolver404:
+        return None
+
+    args = resolved.args
+    if len(args) != 2 or not all([a.isnumeric() for a in args]):
```

Comment:
Why !=2? Need to add comments to know what this is doing.

Reply:
Added a comment.